examples/lr91/causal_test_max_conductances_test_suite.py

```diff
@@ -0,0 +1,190 @@
+import pandas as pd
```

Comment:
Ahh, thanks for pointing this out, it does need to be documented better. 

`causal_test_max_conductances.py` is the original example which has only been updated to function with the new `base_test_case` functionality.  

`causal_test_max_conductances_test_suite.py` makes use of the new `test_suite` functionality rather than using custom loops like the original. 

I think having both for comparison may be useful to show why the test_suite is more concise? I'll add a readme explaining the differences!